pandas/core/reshape/reshape.py

```diff
@@ -166,7 +170,7 @@ def _make_selectors(self):
 
         comp_index = ensure_platform_int(comp_index)
         stride = self.index.levshape[self.level] + self.lift
-        self.full_shape = ngroups, stride
+        self.full_shape = ngroups, int(stride)  # int() for mypy
```

Comment:
we probably don't want to be casting numpy integer types to python ints just to appease mypy. We probably need an int-like alias to use where we currently use int in type annotations.

I think better to leave this unchanged until we've had that discussion.

Reply:
sounds good, will revert